ext/nnstreamer/tensor_filter/tensor_filter_cpp.cc

```diff
@@ -241,6 +237,13 @@ void fini_filter_cpp (void) __attribute__ ((destructor));
 void
 init_filter_cpp (void)
 {
+  NNS_support_cpp.allow_in_place = FALSE;      /** @todo: support this to optimize performance later. */
```

Comment:
Should this be changed with `extern C` wrapper as C supports non-trivial designated initializers.

Reply:
No. it's ok with init_filter_cpp. If you are concerned with the readability, you may move init_filter_cpp to Line 55.

Reply:
Moved to line 55 for readability.